### ISSUE TITLE
Suggestion: Adding a comment to an instruction

### ISSUE THREAD

**Issue by pierreay** [shioyadan/Konata#3]
Hi,

Like in a disassembler, we work with hundreds, thousands, even millions lines of code in a simple window. And like in main disassembler, it will be super helpful to have the possibility to annotate the code, with some kinds comments, for example in the left-panel.

This is just a suggestion. Hope it's appreciate!

P.S. : Thanks for the great work, Konata is very useful!

**Comment by shioyadan** [comment]
Thank you for your suggestion!
I'm very grad that Konata is useful for you.

Do you mean features that add comments to disassembled instructions on the left-panel or stages on the right panel? 
I think that such features are attractive, so I will consider and implement it. 

Konata has a bookmark feature that can be used from the right-click menu, which is not enough for your request, but I think it may be somewhat useful in a situation when you feel that you need the comment feature.

**Comment by pierreay** [comment]
Yes, my first thought was about "add comments to disassembled instructions on the left-panel". Currently, I don't see a particular use case for adding comments to a particular stage on the right panel, but maybe it can be useful for someone.

In my particular use case, when I have a complicated code with multiple loops and a lot of iterations, sometimes I have to trace manually the control flow of the code: adding comments will be very useful when we spot, for e.g., beginning/end of a function, beginning/end of a particular loop, a sought-after iteration, etc.

Yeah, the bookmark feature is helpful sometimes, but it doesn't cover all the situations covered by comments (and vice-versa). However, it already helps a little for me!

Thank you for your consideration!

P.S.: If you prefer to close the "issue" even if the feature is not already implemented, feel free to do so.
